images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -18,6 +18,12 @@ function Get-BazeliskVersion {
     return "Bazelisk $bazeliskVersion"
 }
 
+function Get-BicepVersion {
```

Comment:
The Bicep  version is missing:

![image](https://user-images.githubusercontent.com/47745270/122413292-eccb8a80-cf8e-11eb-83ee-c5624dd058ea.png)


https://github.visualstudio.com/virtual-environments/_build/results?buildId=111916&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=72559e73-928c-58e0-b67e-e22d7a78faae
https://github.visualstudio.com/virtual-environments/_build/results?buildId=111915&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=72559e73-928c-58e0-b67e-e22d7a78faae

Reply:
Updates pushed. Thanks!